src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1453,5 +1483,336 @@ public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
+          ""kind"": ""function""
+        },
+        ""collections::list"": {
+          ""name"": ""list"",
+          ""parentKey"": ""collections"",
+          ""kind"": ""type""
+        },
+        ""collections"": {
+          ""kind"": ""namespace""
+        }
+      },
+      ""results"": [
+        {
+          ""ruleId"": ""C2001"",
+          ""message"": {
+            ""arguments"": [
+              ""ptr""
+            ]
+          },
+          ""ruleMessageId"": ""default"",
+          ""analysisTarget"": {
+            ""uri"": ""file:///home/buildAgent/src/collections/list.cpp""
+          },
+          ""locations"": [
+            {
+              ""physicalLocation"": {
+                ""fileLocation"": {
+                  ""uri"": ""file:///home/buildAgent/src/collections/list.h""
+                },
+                ""region"": {
+                  ""startLine"": 1,
+                  ""startColumn"": 1,
+                  ""endLine"": 1,
+                  ""endColumn"": 28,
+                  ""length"": 27,
+                  ""snippet"": {
+                    ""text"": ""add_core(ptr, offset, val);""
+                  }
+                }
+              },
+              ""fullyQualifiedLogicalName"": ""collections::list::add""
+            }
+          ],
+          ""codeFlows"": [
+            {
+              ""message"": {
+                ""text"": ""Path from declaration to usage""
+              },
+              ""threadFlows"": [
+                {
+                  ""locations"": [
+                    {
+                      ""step"": 1,
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///home/buildAgent/src/collections/list.h""
+                          },
+                          ""region"": {
+                            ""startLine"": 15,
+                            ""snippet"": {
+                              ""text"": ""int *ptr;""
+                            }
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""collections::list:add"",
+                        ""message"": {
+                          ""text"": ""Variable \""ptr\"" declared.""
+                        }
+                      },
+                      ""kind"": ""Declaration"",
+                      ""module"": ""platform"",
+                      ""importance"": ""essential"",
+                      ""properties"": {
+                        ""sarifv1/annotations"": null,
+                        ""sarifv1/essential"": false,
+                        ""sarifv1/fullyQualifiedLogicalName"": ""collections::list:add"",
+                        ""sarifv1/id"": 0,
+                        ""sarifv1/logicalLocationKey"": null,
+                        ""sarifv1/taintKind"": 0,
+                        ""sarifv1/target"": null,
```

Comment:
We need to test that this works for non-null values of `target`, `targetKey`, and `values`.

Reply:
... unless and until we put the whole run in the property bag! I'm going to ping Michael on this.

Reply:
Done